__tests__/app.test.js

```diff
@@ -53,3 +52,39 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 sends an article matching the article_id.", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        expect(article.article_id).toBe(2);
+      });
+  });
+  test("Article should be of the correct format.", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .then(({ body }) => {
+        const article = body.article;
+        expect(article).toHaveProperty("author");
```

Comment:
We could make this more thorough by checking the type of the property value because currently we only know that it has the keys we would expect.